api/src/test/java/org/pmiops/workbench/api/CohortsControllerTest.java

```diff
@@ -440,61 +479,185 @@ public void testCreateCohortBadCriteria() {
   }
 
   @Test
-  public void testUpdateCohort() throws Exception {
-    Cohort cohort = createDefaultCohort();
-    cohort =
+  public void testUpdateCohortAccessLevelOwner() {
+    Cohort saved = createTestCohort(WorkspaceAccessLevel.WRITER);
+    // update
+    Cohort toUpdate = saved.name(UPDATED_COHORT_NAME);
+    CLOCK.increment(1000); // lets say time ticked 1 sec past
+    cohortsController
+        .updateCohort(workspace.getNamespace(), workspace.getId(), toUpdate.getId(), toUpdate)
+        .getBody();
+    Cohort updated =
         cohortsController
-            .createCohort(workspace.getNamespace(), workspace.getId(), cohort)
+            .getCohort(workspace.getNamespace(), workspace.getId(), saved.getId())
             .getBody();
```

Comment:
Unnecessary call. cohortsController#updateCohort will return the updated cohort.